crates/subspace-networking/src/node.rs

```diff
@@ -399,6 +399,14 @@ impl Node {
     pub async fn get_closest_peers(
         &self,
         key: Multihash,
+    ) -> Result<impl Stream<Item = PeerId>, GetClosestPeersError> {
+        self.get_closest_peers_internal(key).await
+    }
+
+    /// Get closest peers by multihash key using Kademlia DHT.
+    async fn get_closest_peers_internal(
```

Comment:
This looks like a leftover from refactoring, it doesn't seem to be necessary

Reply:
It's part of the "request batching" solution for request semaphore.

Reply:
What I mean is that this method is identical to the public method, hence I don't see why it needed to be introduced, could have left it as is.